webdriver-spec.html

```diff
@@ -5047,9 +5052,29 @@
    <dd><p>Toggle <var>element</var>’s <a>selectedness</a> state.
 
    <dt><a><code>option</code> element</a>
-   <dd><p>Set <var>element</var>’s <a>selectedness</a> state to true.
+   <dd>
+     <ol>
+       <li><p>Let <var>currently selected</var> is the <a>container</a> with
+         a <a>selectedness</a> state of true.
+
+       <li><p>Let <var>parent node</var> be the calling
+         <a><code>parentNode</code></a> on <var>element</var>'s <a>container</a>.
+
+       <li><p>Run the <a>focusing steps</a> on <var>parent node</var>.
 
-    <p class=issue>Ensure all correct events are fired.
+       <li><p><a>Fire</a> an <a><code>input</code> event</a> at <var>parent node</var>.
+
+       <li><p>Set <var>element</var>’s <a>selectedness</a> state to true.
+
+       <li><p>If <var>currently selected</var> does not equal <var>element</var>:
```

Comment:
This seems like a convoluted way of just checking `element`'s "selectedness", especially since this logic seems shaky in a multiple select.